rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -375,25 +380,24 @@ def test_migration_11(
     - Test that detecting arbitrum one accounts works properly
     """
     rotki = rotkehlchen_api_server.rest_api.rotkehlchen
-
+    # set high version so that DB data updates dont't get applied
+    with rotki.data.db.conn.write_ctx() as write_cursor:
+        write_cursor.executemany(
+            'INSERT OR REPLACE INTO settings(name, value) VALUES (?, ?)',
+            [
+                (UpdateType.SPAM_ASSETS.serialize(), 999),
+                (UpdateType.RPC_NODES.serialize(), 999),
+                (UpdateType.CONTRACTS.serialize(), 999),
+                (UpdateType.GLOBAL_ADDRESSBOOK.serialize(), 999),
+            ],
+        )
     migration_patch = patch(
         'rotkehlchen.data_migrations.manager.MIGRATION_LIST',
         new=[MIGRATION_LIST[5]],
     )
 
     arbitrum_one_addresses = [ethereum_accounts[1]]
-    with ExitStack() as stack:
-        setup_evm_addresses_activity_mock(
```

Comment:
no need to use both VCR and mocking evm queries. Only kind of hard thing was to find an address that at the time of the test was active **only** in mainnet and arbitrum. Did it manually by searching